src/styles/index.scss

```diff
@@ -1,3 +1,117 @@
+html {
+  font-family: roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+}
+
 body {
   margin: 0;
+  display: block;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background: #fff;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+  display: block;
+}
```

Comment:
You can move in in card and use `&__img`  selector. 
Check all case when you can use & in scss code)